src/elements/mod.rs

```diff
@@ -44,33 +44,19 @@ impl From<&TxIn> for IssuanceValue {
         };
 
         IssuanceValue {
-            asset_id: asset_id.to_hex(),
-            asset_entropy: asset_entropy.to_hex(),
-            contract_hash: contract_hash.map(|h| h.to_hex()),
+            asset_id,
+            asset_entropy,
+            contract_hash,
             is_reissuance,
             asset_blinding_nonce: if is_reissuance {
-                Some(hex::encode(issuance.asset_blinding_nonce.as_ref()))
+                Some(issuance.asset_blinding_nonce)
             } else {
                 None
             },
-            assetamount: match issuance.amount {
-                Value::Explicit(value) => Some(value),
-                Value::Null => Some(0),
```

Comment:
This converts now to `None`, but I think it's more appropriate

Reply:
When is this `None`? My `check-api-stablity` script didn't catch this because it appears to typically be either `Explicit` or `Confidential`

Reply:
`tokenamount` is `Null` in a issuance without reissuance token.

Adding 

    asset3=$(cli issueasset 10 0 false)

should show the breaking change

e.g. here it is 0, https://blockstream.info/liquid/api/tx/5221eb18850f88955214950c13fbff766bd1b33743099b6c05a504407585401a

Reply:
Thanks!

I added a test for this in 4a2d465f1704a71e531fd31eb9143aae82973a03. It now detects the incompatibility:

```
Checking GET /tx/f92f1f6638d4f3304ad28a0287206d552592f2678b719f2df9b552d4d021b2a3 ...
--- /dev/fd/63  2024-02-08 13:11:18.129749079 +0200
+++ /dev/fd/62  2024-02-08 13:11:18.129749079 +0200
@@ -18,4 +18,3 @@
         "contract_hash": "0000000000000000000000000000000000000000000000000000000000000000",
-        "is_reissuance": false,
-        "tokenamount": 0
+        "is_reissuance": false
       },
```

It's kind of weird to me that elements represents this as a null rather than an explicit 0, but I guess that it makes sense for us to follow suit.